testing/redisfixture/redisfixture.go

```diff
@@ -73,8 +79,11 @@ func checkRedisConnection(ctx context.Context, t types.TestingTB, client *redis.
 	switch {
 	case err != nil && err.Error() == "ERR DB index is out of range":
 		assert.Assert(t, err)
-	case err != nil:
-		t.Skip("Redis not available")
+	case err != nil && !mustRunAllTests:
```

Comment:
Hmm - this line seems like a bug. It probably shouldn't have changed

Reply:
Think you might be right. It shouldn't have the `!mustRunAllTests` I think

Reply:
I'll put up another PR to fix but I think it will need fixing in `backplane-go` too (this code was just lifted straight from there)